jena-arq/src/main/java/org/apache/jena/riot/RDFDataMgr.java

```diff
@@ -1061,17 +1059,15 @@ public static void writeQuads(OutputStream out, Iterator<Quad> iterator) {
      * @param lang Language
      * @param baseIRI Base IRI
      * @return Iterator over the triples
+     * @deprecated Use {@link AsyncParser#asyncParseTriples} or for N-Triples, {@link RiotParsers#createIteratorNTriples}
      */
+    @Deprecated
```

Comment:
Does this need to be deprecated, since it's updated to point at the new code AND it handles the NTriples special case it seems nice to keep this method.  Maybe just update the Javadoc to note that this potentially spawns a thread and provide a pointer to `AsyncParser`?

Reply:
Added javadoc about a thread. 

I'd like the option to remove it; there is a fixed choice built-in. The app may not want NT/NQ done "same thread". Deprecation is the only way to let users/applications know that the async parser is now available. The javadoc does refer to `RiotParsers.createIteratorNQuads`.

Reply:
Agreed